api/v1alpha1/hostedcluster_types.go

```diff
@@ -23,7 +23,7 @@ type HostedClusterSpec struct {
 	// workers. It should have an ".dockerconfigjson" key containing the pull secret JSON.
 	PullSecret corev1.LocalObjectReference `json:"pullSecret"`
 
-	SigningKey corev1.LocalObjectReference `json:"signingKey"`
+	OIDCRouteTLSSecret corev1.LocalObjectReference `json:"oidcRouteTLSSecret"`
```

Comment:
@ironcladlou yeah, ugh.  This is the part I'd like to do something different.  I don't want this Route secret on the HC spec.  I just want the `IssuerURL`.  I just don't know how to achieve that when IAM setup must precede the HC creation.